app/controllers/repositories_controller.rb

```diff
@@ -60,6 +60,10 @@ def variant
     _variant
   end
 
+  def option
```

Comment:
No problem.  It's midnight here in Tokyo, so I'll make the change in the morning ... ~9 hours from now.